src/stack_mitosis/sudo.clj

```diff
@@ -60,6 +60,8 @@
     provider))
 
 (comment
+  ;; Make sure AWS_CONFIG_FILE, AWS_CREDENTIAL_PROFILES_FILE are both set in
+  ;; environment or the appropriate aws key and secret are present.
```

Comment:
🎨 Maybe good to add to set up instructions as well

Reply:
Added some, let me know if it helps.